docs/The-Free-Plan.md

```diff
@@ -0,0 +1,70 @@
+---
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# What is the Free Plan?
+The free plan is ideal for start-ups and small businesses to manage expenses. With the Free Plan, a workspace admin can set up their team with Expensify Cards, reimburse cash expenses, send invoices, and manage bills, all for free! You will have total visibility and control over all spending associated with your workspace in real time.
+
+# Features Included with the Free Plan
+- Expensify Cards for all employees
+- Invoicing
+- Bill Pay
+- Unlimited receipt scanning for everyone in the company
+- Free next-day ACH reimbursements for cash expenses
+- Up to 1% cash back for card spending over $25,000 per month
+- Free corporate travel booking
+
+# Setting Up the Free Plan
+- Navigate to new.expensify.com, enter your company email address, and set a password
+- Click the **green “+”** button and select **_New workspace_**
+
+<img class="image" src="/assets/images/new-workspace.png">
```

Comment:
The images also look good. I noticed I forgot to include in the instructions that you can use the property [width](https://www.w3schools.com/tags/att_img_width.asp) to adjust the size of the image. For example, this is how we can set the width to `400px` (the height will be adjusted automatically in proportion to the `width` value): 

<img width="333" alt="Screen Shot 2022-10-19 at 18 25 48" src="https://user-images.githubusercontent.com/6829422/196823239-cc907d9d-19be-456b-8e12-8e02d337bf13.png">

Also, I noticed we the images are left-aligned by default (I'm going to create a PR to center them):

<img width="1650" alt="Screen Shot 2022-10-19 at 18 33 56" src="https://user-images.githubusercontent.com/6829422/196823924-eab788cd-1e52-486d-9749-2a3112a6089e.png">